lib/modules/manager/cnb/extract.ts

```diff
@@ -0,0 +1,108 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { regEx } from '../../../util/regex';
+import { getDep } from '../dockerfile/extract';
+import type {
+  ExtractConfig,
+  PackageDependency,
+  PackageFileContent,
+} from '../types';
+import { type ProjectDescriptor, ProjectDescriptorToml } from './schema';
+
+const dockerPrefix = regEx(/^docker:\/?\//);
+const buildpackRegistryRef = regEx(/^[a-z0-9\-.]+\/[a-z0-9\-.]+(?:@.+)?$/);
```

Comment:
is this a valid docker image for buildpacks:

```toml
[[io.buildpacks.group]]
uri = "node@sha256:914458d8617650599ec2c05f6754403a1ce08cb471b0b1de1de439c539f8d45f"
[[io.buildpacks.group]]
uri = "node:latest@sha256:914458d8617650599ec2c05f6754403a1ce08cb471b0b1de1de439c539f8d45f"
```

it's a valid docker reference